src/content/en/fundamentals/primers/service-workers/the-players.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: shared/narrow
+published: false
+title: "The Players"
```

Comment:
I _really_ don't like the term "players" in this context. I don't know that there's a better term to use instead. Could you just say that there are service workers and clients without trying to assign a single term to cover both of them?